docs/users/security/safu.md

```diff
@@ -0,0 +1,93 @@
+<!--
+order: 3
+-->
+
+# Simple Arrangement for Funding Upload (SAFU)
+
+Learn about the Simple Arrangement for Funding Upload (SAFU)
+on Evmos {synopsis}
+
+## Overview
+
+<!-- markdown-link-check-disable-next-line -->
+The [Simple Arrangement for Funding Upload (the "SAFU")](https://github.com/evmos/evmos/tree/main/docs/users/security/SAFU.pdf)
+outlines the post-exploit policy for active vulnerabilities
+in the Evmos blockchain. The SAFU is intended for white hat hackers
+and outlines the process for returning funds and calculating rewards
+for vulnerabilities found in the network.
+In summary, the SAFU states the following:
+
+* Hackers are not at risk of legal action if they act in accordance
+  with the SAFU.
+* Hackers have 48 hours to return any exploited funds
+  to a specified Dropbox address and can claim a reward of
```

Comment:
tbh I'm not too happy with calling this "Dropbox" address - this written as one word with a capital D is a trademarked name of a company, better would be to use e.g. "drop box"?